apps/ledger-live-desktop/src/storyly/StorylyProvider.tsx

```diff
@@ -58,8 +64,15 @@ const StorylyProvider: React.FC<StorylyProviderProps> = ({ children }) => {
     storylyRef.current?.on("closeStoryGroup", clear);
 
     storylyRef.current?.openStory({ group: query?.g, story: query?.s, playMode: query?.play });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Can't we just add language to the dependencies ?

Reply:
I agree with Lucas, we shouldn't lie to react (https://overreacted.io/a-complete-guide-to-useeffect/#:~:text=Don%E2%80%99t%20Lie%20to%20React%20About%20Dependencies)

Reply:
if we add it at this level, it will trigger an init each and every time , it's not recommended

Reply:
language is not a composite dependency to update the init process each and every time.